rero_ils/manual_translations.txt

```diff
@@ -43,11 +43,11 @@ _('ui_language_de')
 _('ui_language_it')
 
 # provisionActivity
-_('bf:Publication')
-_('bf:Manufacture')
-_('bf:Distribution')
-_('bf:Production')
-_('bf:Place')
+#_('bf:Publication')
```

Comment:
If you add this here and yo change it in the documents JSON schema we have to translate it twice !
bf:Publication -> to translate
Publication -> to translate

Reply:
You're right, I wanted to delete all this strings but forgot one. Thanks.